src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+    height: 80px;
+    padding: 12px 37px;
+    background-color: #fff;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    img {
```

Comment:
These two floats are needed because they have their parent is `flexbox` with `justify-content: space-between`